pkgs/applications/editors/brackets/default.nix

```diff
@@ -9,55 +11,56 @@ let
     ];
   };
 in
-stdenv.mkDerivation rec {
-  name = "brackets-${version}";
-  version = "1.8";
-
-  src = fetchurl {
-    url = "https://github.com/adobe/brackets/releases/download/release-${version}/Brackets.Release.${version}.64-bit.deb";
-    sha256 = "0b2k0vv1qwmsg1wckp71yrb86zp8zisskmzzvx9ir19bma9jzr42";
-    name = "${name}.deb";
-  };
+  stdenv.mkDerivation rec {
```

Comment:
Fixed it, seen what you were talking about; and also I do not know what is going on with git rebase with me lately, I can't squish the commits. I asked in IRC but no one answered